pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <ddf.version>2.13.3</ddf.version>
+        <ddf.version>2.13.4-SNAPSHOT</ddf.version>
```

Comment:
❓ Does this bring along anything else we need to be concerned about or need to test against?

Reply:
There have been a handful of commits to DDF since 2.13.3 was released, but none of them look big or risky to me, from a downstream perspective.